lib/blueberry_rails/app_builder.rb

```diff
@@ -4,12 +4,15 @@ class AppBuilder < Rails::AppBuilder
     include BlueberryRails::ActionHelpers
 
     def readme
-      template 'README.md.erb', 'README.md'
+      copy_file 'README.md', 'README.md'
     end
 
-    def replace_gemfile
-      remove_file 'Gemfile'
-      template 'Gemfile_custom.erb', 'Gemfile'
+    def gitignore
+      copy_file 'gitignore_custom', '.gitignore'
+    end
+
+    def gemfile
+      template 'Gemfile_custom', 'Gemfile'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.